scenes/MosaikHeer.scn

```diff
@@ -926,7 +926,7 @@
 /ch/28/mix/16 OFF   -oo
 /ch/28/grp %00000000 %000000
 /ch/28/automix OFF  +0.0
-/ch/29/config "Ambnt L" 1 YEi 3
+/ch/29/config "Ambnt L" 1 YEi 5
```

Comment:
Actually, we can get rid of the "Ambient mics". I haven't used them for a while and I don't think they bring much value. What do you think? Let's bring them back to standard settings. I think best is to copy the settings from channel 28 to 29 and 30. Make sure the inputs are set back to 29 and 30 as well (they are 5 and 6 at the moment).

Reply:
Agreed and changed.